runtime/ast/composite.go

```diff
@@ -280,6 +282,11 @@ func (d *CompositeDeclaration) ConformanceList() []*NominalType {
 	return d.Conformances
 }
 
+func (d *CompositeDeclaration) IsResourceDestructionDefaultEvent() bool {
```

Comment:
This should maybe be moved to `sema`

Reply:
Just from a syntax vs semantics point-of-view. Syntactically there is nothing special about this declaration, but semantic analysis then deems it special, based on its name.

Feel free to leave here, not a big deal.